packages/tomls/src/oracle_adapters/configs/init_lm_token_prices.toml

```diff
@@ -1,11 +1,41 @@
+[invoke.oracle_adapters_reya_lm_selini_init_price_range]
+target = ["OracleAdaptersProxy"]
+fromCall.func = "owner"
+func = "setLmTokenPriceConfiguration"
+args = [
+    "<%= settings.reyaLmAssetPairIdSELINI %>",
+    { priceLowerBound = "<%= parseEther('0.9') %>", priceUpperBound = "<%= parseEther('2') %>" },
+]
+depends = ["invoke.oracle_adapters_reya_lm_price_updater_owner"]
+
+[invoke.oracle_adapters_reya_lm_amber_init_price_range]
+target = ["OracleAdaptersProxy"]
+fromCall.func = "owner"
+func = "setLmTokenPriceConfiguration"
+args = [
+    "<%= settings.reyaLmAssetPairIdAMBER %>",
+    { priceLowerBound = "<%= parseEther('0.9') %>", priceUpperBound = "<%= parseEther('2') %>" },
```

Comment:
worth putting these values in variables?

Reply:
this is good point. however these are pretty much constant, so I'm going to leave it as it is for now